src/components/ManageSprints.vue

```diff
@@ -74,47 +93,41 @@
     data: () => ({
       title: "Manage Sprints",
       newSprint: {
-        id: -1,
+        id: 0,
+        sprintNumber: 0,
+        name: "",
         startDate: "",
         endDate: ""
-      },
-      showModal: false
+      }
     }),
 
     props: {
       project: {
         type: Object,
         required: true
-      },
-      sprints: {
-        type: Array,
-        required: true
       }
     },
 
     computed: {
-      actions: function() {
-        return [
-          {
-            name: 'Save',
-            class: 'btn-primary',
-            action: this.save
-          }
-        ]
-      }
-    },
-
-    created() {
-      if (this.$route.params.showModal === true) {
-        this.showModal = true;
+      minimumSprint: function() {
+        const currentSprint = _.findLast(_.orderBy(this.project.sprints, "sprintNumber", "asc"));
+        return parseInt(currentSprint.sprintNumber) + 1;
       }
     },
 
     methods: {
       save() {
-        this.newSprint.id = this.sprints.length;
-        this.newSprint.startDate = document.getElementById('startDate').val;
-        console.log('Save sprint', this.newSprint);
+        this.newSprint.name = "Sprint " + this.newSprint.sprintNumber;
+        this.newSprint.id = _.camelCase(this.newSprint.name);
+        this.newSprint.teams = _.forEach(this.project.teams, team => team.criteria_values = [2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2]);
```

Comment:
I don't like how I have set the default values of the criteria_values, might be worth having a constants file? @chartley94

Reply:
A constants file is something to think about for sure. I'd keep it like this for now - or if you want, create a computed property called `defaultCriteriaValue` that returns and generates an array, giving each criteria the default value.

Also, the default value should be 0 if I remember correctly. This will show just an empty circle on the health check UI.